src/components/Navbar.jsx

```diff
@@ -1,22 +1,22 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
+import { FaUser } from 'react-icons/fa6';
 
 const Navbar = () => (
-  <div className="Navbar">
-    <header>
-      <nav>
-        <div className="logo">Bookstore CMS</div>
-        <ul>
-          <li>
-            <Link to="/">Books</Link>
-          </li>
-          <li>
-            <Link to="/Categories"> Categories</Link>
-          </li>
-        </ul>
-      </nav>
-    </header>
-  </div>
+  <header>
+    <nav>
+      <div className="logo">Bookstore CMS</div>
+      <ul>
+        <li>
+          <Link to="/">Books</Link>
+        </li>
+        <li>
+          <Link to="/Categories"> Categories</Link>
+        </li>
```

Comment:
- [ ] Kindly adhere to the project requirement because as I can see you did an awesome job by following the design but not properly, You should make your `Menus` in uppercase letters and also add the `active` page class properly, which actually tell the user, on which page the `user`. 

### Requirement
![Screenshot from 2023-09-09 21-59-56](https://github.com/mulugithub/bookstore/assets/66476329/362a2acf-5478-4303-8f0b-8712be4696eb)

### Design Comparison
<table>
<tr>
<th>Zeplin's Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-09-09 22-06-47](https://github.com/mulugithub/bookstore/assets/66476329/0dccbaa4-cb4d-44b5-8aa7-13d6df97f4fd)


</td>
<td>

![Screenshot from 2023-09-09 22-06-32](https://github.com/mulugithub/bookstore/assets/66476329/be37320a-a03b-42b6-b480-073266faccb1)

</td>
</tr>
</table>